MNE/disp3D/3DObjects/digitizer/digitizertreeitem.cpp

```diff
@@ -93,6 +93,15 @@ DigitizerTreeItem::DigitizerTreeItem(int iType, const QString& text)
 
 DigitizerTreeItem::~DigitizerTreeItem()
 {
+    setVisible(false);
+
+    delete m_pRenderable3DEntity;
+
+    for(int i = 0; i < m_lSpheres.size(); ++i) {
+        delete m_lSpheres[i];
+    }
+
+    m_lSpheres.clear();
```

Comment:
If those opbjects are QObjects - consider using QPointer instead of raw pointers - they would be automatically deleted